base/util.jl

```diff
@@ -459,7 +459,7 @@ function getpass end
 
 if Sys.iswindows()
 function getpass(prompt::AbstractString)
-    print(prompt, ':')
+    print(prompt, ": ")
```

Comment:
(Honestly, it seems a bit weird to me that we add a colon at all.   Can't we let the caller decide this?   For example, what if they want to display a prompt `"True (y/n)? "` or `"Enter password here => "`?